.circleci/config.yml

```
@@ -0,0 +1,154 @@
+aliases:
+  - &credentials
+    name: Retrieve temporary Algolia credentials if needed
+    command: |
+      if [ -z "ALGOLIA_ADMIN_KEY_MCM" ]; then
+        curl -s https://algoliasearch-client-keygen.herokuapp.com | sh >> $BASH_ENV
+      fi
+
+  - &check_bundler
+    name: Which bundler?
+    command: bundle -v
+
+  - &restore_cache
+    name: Restore Bundler cache
+    keys:
+      - rails-cache-<< parameters.version >>-<< parameters.rails-version >>-<< parameters.sequel-version >>-{{ checksum "Gemfile" }}
+      - rails-cache-<< parameters.version >>-<< parameters.rails-version >>-<< parameters.sequel-version >>-
+
+  - &install_bundler
+    name: Bundle Install
+    command: bundle check || bundle install
+
+  - &save_cache
+    name: Save Bundler cache
+    key: rails-cache-<< parameters.version >>-<< parameters.rails-version >>-<< parameters.sequel-version >>-{{ checksum "Gemfile" }}
+    paths:
+      - vendor/bundle
+
+  - &run_tests
+     name: Run unit and integration tests
+     command: |
+       bundle exec rake
+
+references:
+  default_docker_ruby_executor: &default_docker_ruby_executor
+    image: circleci/ruby:<< parameters.version >>
+    environment:
+      BUNDLE_JOBS: 3
+      BUNDLE_RETRY: 3
+      BUNDLE_PATH: vendor/bundle
+      RAILS_VERSION: << parameters.rails-version >>
+      SEQUEL_VERSION: << parameters.sequel-version >>
+
+version: 2.1
+
+jobs:
+  tests_rails_3:
+    description: Build, unit and integration tests for Ruby
```

Can you remove this for the other job descriptions too, please?
```suggestion
    description: Build, unit and integration tests
```